test/routes/src/server/routes/t_github-auth.js

```diff
@@ -157,16 +158,6 @@ describe('The login route', () => {
           );
         });
 
-        it('should clear cached organization information', async () => {
```

Comment:
Cache is cleared, but because we fetch list of orgs immediately (to add to user object) cache is re-populated immediately with new data.